plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/metastore/glue/TestDeltaLakeGlueMetastore.java

```diff
@@ -241,6 +252,37 @@ public void testHideNonDeltaLakeTable()
                 .isEmpty();
     }
 
+    @Test
+    public void testDoubleCreateSchemaForTheSameSession()
+    {
+        String testSchemaName = "test_schema_" + randomName();
+
+        DeltaLakeMetadata metadata = metadataFactory.create(session.getIdentity());
+
```

Comment:
try/finally to ensure that the Glue database does get deleted even in case of exceptions